content/applications/general/users.rst

```diff
@@ -32,7 +31,6 @@ Fill in the form with all the required information. Under the :doc:`Access Right
 The list of applications shown is based on the applications installed on the database.
 
 .. image:: users/new-user.png
-   :align: center
    :alt: View of a user's form emphasizing the access rights tab in Odoo.
 
 After filling out all the necessary fields on the page, manually :guilabel:`Save`. An invitation
```

Comment:
There is no button with "Save" text, rather the _Save manually_ cloud icon is used here
```suggestion
After filling out all the necessary fields on the page, :icon:`fa-cloud-upload` :guilabel:`(Save
manually)`. An invitation email is automatically sent to the user, using the email in the
:guilabel:`Email Address` field. The user must click on the link included in the email to accept the
invitation, and to create a database login.
```